cmd/root.go

```diff
@@ -111,6 +114,34 @@ func PreRun(cmd *cobra.Command, args []string) {
 func Run(c *cobra.Command, names []string) {
 	filter := filters.BuildFilter(names, enableLabel)
 	runOnce, _ := c.PersistentFlags().GetBool("run-once")
+	httpAPI, _ := c.PersistentFlags().GetBool("http-api")
```

Comment:
To promote safe practices, I think we should do a fatal exit in case the token is unset/empty.

Reply:
I believe you are referring to `apiToken, _ := c.PersistentFlags().GetString("http-api-token")` instead of `httpAPI, _ := c.PersistentFlags().GetBool("http-api")`. And I agree with you. I'll put a conditional to validate it.